src/analog/commands/downlink/setTime2000.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * Command used to set time on device.
+ *
+ * @example
+ * ```js
+ * import SetTime2000 from 'jooby-codec/analog/commands/downlink/SetTime2000.js';
+ *
+ * // 240 seconds to the future
+ * const parameters = {sequenceNumber: 45, seconds: -120};
+ * const bytes = correctTime2000.toBytes(parameters);
+ *
+ * // command binary representation
+ * console.log(bytes);
+ * // [2, 5, 78, 43, 189, 152, 173]
+ * ```
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/analog/commands/SetTime2000.md#request)
+ */
+
+import {TCommandId, TBytes, TUint8, TInt32} from '../../../types.js';
+import BinaryBuffer, {IBinaryBuffer} from '../../../utils/BinaryBuffer.js';
+import * as command from '../../utils/command.js';
+
+
+interface ISetTime2000Parameters {
+    /** unique time manipulation operation number */
+    sequenceNumber: TUint8;
+    /** seconds */
+    seconds: TInt32;
+}
+
+
+export const id: TCommandId = 0x02;
+export const headerSize = 2;
+const COMMAND_BODY_SIZE = 5;
+
+export const examples = {
+    'set time to 2023.04.03 14:01:17 GMT': {
+        id,
+        headerSize,
+        parameters: {
+            sequenceNumber: 78,
+            seconds: 733845677
+        },
+        bytes: [0x02, 0x05, 0x4e, 0x2b, 0xbd, 0x98, 0xad]
```

Comment:
would be nice to split it to two lines

and in some other places as well

Reply:
fixed